clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -3318,6 +3350,20 @@ llvm::Constant *CodeGenFunction::EmitCheckTypeDescriptor(QualType T) {
       DiagnosticsEngine::ak_qualtype, (intptr_t)T.getAsOpaquePtr(), StringRef(),
       StringRef(), std::nullopt, Buffer, std::nullopt);
 
+  if (IsBitInt) {
+    // The Structure is: 0 to end the string, 32 bit insigned integer in target
+    // endianness, zero.
```

Comment:
```suggestion
    // The Structure is: 0 to end the string, 32 bit unsigned integer in target
    // endianness, zero.
```
typo?

Reply:
Yes. OMG. I should have noticed. I is close to U on my keyboard.